api/src/main/java/io/opencensus/tags/Tag.java

```diff
@@ -28,18 +29,39 @@
 @AutoValue
 public abstract class Tag {
 
+  private static final TagMetadata METADATA_UNLIMITED_PROPAGATION =
```

Comment:
Should we expose this in the API? Otherwise users need to re-create this object in their code.

Reply:
agree on exposing it.

Reply:
SG, updated.

Reply:
I think we should not because once you have another property in the metadata you need to expose a lot of constants correct?

Reply:
Removed from public APIs. If they're really needed by users we can always expose them later.